pageserver/src/tenant/timeline.rs

```diff
@@ -5442,6 +5459,36 @@ impl Timeline {
 
         Ok(())
     }
+
+    #[cfg(test)]
+    /// Return all keys at the LSN in the image layers
```

Comment:
Sort of styleguide matter, but I wish rustdoc/fmt would handle this, please turn the lines around:

```suggestion
    /// Return all keys at the LSN in the image layers
    #[cfg(test)]
```

Rationale: imagine the comment was large, no one would have any clue what attributes had been applied to it *above* it.